seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2417,11 +2417,11 @@
         "",
         "// Only change code below this line",
         "",
+        "",
+        "// Only change code above this line",
+        "timesFive(5);",
```

Comment:
Although it is good to introduce people to the function call, this function call may confuse people because nothing else in the editor is `timesFive(5)`. Plus when the page loads, this will cause an error.

![image](https://cloud.githubusercontent.com/assets/2754821/16901352/cd47e66c-4bf5-11e6-8d99-1d602d13271a.png)

So what I would do is

- remove the `timesFive(5)` function call,
- add a `minusSeven(10)` function call below the `minusSeven` function initialization as an example of how to call a function, and 
- add to the instructions something along the lines of you can call the `timesFive` function to test it.